src/main/java/racingcar/controller/Game.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.controller;
+
+import racingcar.game.Cars;
+import racingcar.game.validator.InputValidator;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.stream.IntStream;
+
+import static racingcar.controller.constants.IntegerConstants.*;
+
+public class Game {
+    public Integer round;
+    public Cars cars;
+
+    public void run(){
+        askCarNames();
+        askGameRounds();
+        racing();
+        getWinners();
+    }
+
+    public void askCarNames(){
+        OutputView.askCarNames();
+        cars = new Cars(InputView.readCarNames());
+    }
```

Comment:
해당 메서드는 `자동차 이름을 입력받는 메서드`인 것 같아요!

1) 자동차 이름을 입력받기
2) 입력받은 자동차 이름을 토대로 car 객체 생성하기

위와 같이 2개의 일을 처리하고있는 것 같은데, 각각으로 분리하는 것에 대해서는 어떻게 생각하시나요?
`askCarNames` 라는 메서드명에서는 `car 객체를 생성한다` 라는 추측이 되지 않아서 여쭈어봅니다!

Reply:
좋은 의견 감사합니다. 맞네요! 제가 해당 미션을 풀 때 집중 했었던 것은
1. 자동차 이름 입력
2. 랜덤 값 할당
3. 결과 출력

부분으로 나뉜다는 것을 생각하고 이렇게 메서드를 분리 했었는데 너무 조그맣게 생각했던 것 같아요. 

그런데 말씀하셨던 것 처럼 해당 메서드를 두가지로 나눈다면 ```OutputView.askCarNames()``` 메서드 자체를 ```run()``` 안에서 바로 호출하게 될 것 같아요. 그러면 가독성 부분에 있어서 떨어지지 않을까 생각합니다. 혹은 메서드 이름을 좀 더 고민해봐야 할까요? 어떻게 생각하시는지 궁금합니다!

Reply:
그렇네요 .. 전체적인 `run()` 에서의 가독성이 좋아지지 않는 것 같네요.
저는 아래와 같은 방법을 생각해보았는데 어떻게 생각하시는지 궁금합니다!

```suggestion
    public void askCarNames() {
        OutputView.askCarNames();
        createCars(InputView.readCarNames());
    }
    
    public void createCars(String carNames) {
        cars = new Cars(carNames);
    }
```

이렇게 `askCarNames()` 에서는 자동차 이름을 묻고 -> `createCars()` 를 호출해서 자동차 객체를 만들어주게 되면 전체적인 `run()` 에서의 가독성은 달라지지 않을 것 같습니다!
근데 이렇게되면 `createCars()` 부분은 `game` 의 역할(게임 진행)이 아니게되므로 다른 클래스내로 이동되어야 할 것 같네요 🤔